.github/workflows/ci_back_build_and_test.yml

```diff
@@ -33,10 +33,12 @@ jobs:
       with:
         dotnet-version: 6.0.x
     - name: Restore dependencies
-      run: dotnet restore src/Core/TicketManager.Core.Api/TicketManager.Core.Api.csproj
+      run: dotnet restore
     - name: Build
-      run: dotnet build src/Core/TicketManager.Core.Api/TicketManager.Core.Api.csproj --no-restore
+      run: dotnet build --no-restore
     - name: Test Domain Tests
-      run: dotnet test tests/Core/TicketManager.Core.DomainTests/TicketManager.Core.DomainTests.csproj
+      run: dotnet test tests/Core/TicketManager.Core.DomainTests/TicketManager.Core.DomainTests.csproj --no-restore
```

Comment:
We could probably add `--no-build` as `dotnet build` no longer targets only app but tests as well